### DISCUSSION TITLE
What is the difference between a fork and a clone in GitHub?

### DISCUSSION THREAD

**Discussion by xectrone** [community/community#131728] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm trying to understand the difference between forking and cloning a repository on GitHub. When would I fork a repository, and when would I clone it? How do these actions affect collaboration and version control, especially when working with others on a project?

**Comment by seynoke** [comment]
When you fork a repository on GitHub, you create a copy of the repository under your GitHub account. This allows you to freely experiment with changes without affecting the original project. Cloning, on the other hand, involves creating a local copy of a repository on your machine, which can then be used to work on the code locally.
